src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -83,25 +88,26 @@ export default function VAFacilityPageV2() {
   const loadingFacilities =
     childFacilitiesStatus === FETCH_STATUS.loading ||
     childFacilitiesStatus === FETCH_STATUS.notStarted;
-  let pageTitle;
-  if (singleValidVALocation) {
-    pageTitle = 'Your appointment location';
-  } else {
-    pageTitle = 'Choose a VA location';
-  }
+  const pageTitle = singleValidVALocation
+    ? 'Your appointment location'
+    : 'Choose a VA location';
+
   const isLoading =
     loadingFacilities || (singleValidVALocation && loadingEligibility);
   const sortFocusEl = 'select';
   const hasUserAddress = address && !!Object.keys(address).length;
 
   useEffect(() => {
-    document.title = `${pageTitle} | Veterans Affairs`;
     dispatch(openFacilityPageV2(pageKey, uiSchema, initialSchema));
   }, []);
 
   useEffect(
     () => {
+      document.title = `${pageTitle} | Veterans Affairs`;
```

Comment:
Moved the document.title assignment to a different useEffect where it will refresh when `singleValidVALocation` boolean is returned otherwise it will display the wrong text in the document title